packages/server/src/common/launch/glsp-server-launcher.ts

```diff
@@ -40,15 +38,15 @@ export abstract class GLSPServerLauncher<T = undefined> {
         return container;
     }
 
-    start(startParams?: T): MaybePromise<void> {
+    start(startParams: T): MaybePromise<void> {
```

Comment:
Is there a reason you removed the optional version of this? I am just wondering because it leads to rather odd code that only overrides this method to provide default values. If we want to keep it maybe we can add a property for default values that need to be provided?